src/api/api.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable linebreak-style */
+
+export async function getPokemonList(url = 'https://pokeapi.co/api/v2/pokemon/') {
+  const apiResp = await fetch(url);
+  const apiRespJson = await apiResp.json();
+  return apiRespJson;
+}
```

Comment:
cuando uno declara una función como async, el resultado si o si va a ser una promesa, es decir que el último await está de más!
```suggestion
export async function getPokemonList(url = 'https://pokeapi.co/api/v2/pokemon/') {
  const response = await fetch(url);
  return response.json();
}
```